data/mods/roulettemons/random-teams.ts

```diff
@@ -0,0 +1,140 @@
+/* eslint max-len: ["error", 240] */
+
+import {PRNG, PRNGSeed} from '../../../sim/prng';
```

Comment:
```suggestion
import {PRNG, PRNGSeed} from '../../../sim/prng';
```
Please don't override eslint errors on a per-file basis. I know TI did this, but it's no longer something we're doing.